library/xml-model/src/main/java/org/cru/godtools/xml/model/Manifest.kt

```diff
@@ -95,7 +95,9 @@ class Manifest : BaseModel, Styles {
     val backgroundImageScaleType: ImageScaleType
 
     @ColorInt
-    val cardBackgroundColor: Int?
+    private val _cardBackgroundColor: Int?
+    @get:ColorInt
+    val cardBackgroundColor: Int get() = _cardBackgroundColor ?: manifest.backgroundColor
```

Comment:
no need for `manifest.` here, you are already in the manifest :)

Reply:
So do I need just have ```val cardBackgroundColor: Int?``` or should I have it be ```DEFAULT_BACKGROUND_COLOR``` instead of ```manifest.backgroundColor```?

Reply:
you do want it to fallback to the manifest background color, but you are already in the manifest object, so there is no need to use `manifest.` to access it

Reply:
Ah okay gotcha!